xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/internal/debug/DebugConfiguration.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package com.xpn.xwiki.internal.debug;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+import org.xwiki.component.annotation.Component;
+import org.xwiki.configuration.ConfigurationSource;
+import org.xwiki.container.Container;
+
+/**
+ * Information about debug configuration.
+ *
+ * @version $Id$
+ * @since 10.6RC1
+ */
+@Component(roles = {DebugConfiguration.class})
+@Singleton
+public class DebugConfiguration
+{
+    @Inject
+    @Named("xwikiproperties")
+    private ConfigurationSource properties;
+
+    @Inject
+    private Container container;
+
+    /**
+     * @return true if resources should be minified when possible
+     * @since 10.6RC1
+     */
+    public boolean isMinify() {
```

Comment:
hmm I'm wondering if this should go in CoreConfiguration instead (which is about the config for oldcore).

Reply:
However seems that `debug.minify` was not set a core configuration property which is maybe an error we did. @tmortagne any take on that?

Reply:
It's not really oldcore configuration. The helper to access it is currently internal but I think it will go in a debug module one day.